presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientConfig.java

```diff
@@ -154,6 +155,7 @@ public void testExplicitPropertyMappings()
                 .put("hive.force-local-scheduling", "true")
                 .put("hive.max-concurrent-file-renames", "100")
                 .put("hive.assume-canonical-partition-keys", "true")
+                .put("mapreduce.input.linerecordreader.line.maxlength", "1MB")
```

Comment:
Maybe pick a more "unnatural" value that could never be a possible default.... like 13MB